Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -1066,9 +1064,6 @@ SamplerState TextureCacheBase::GetSamplerState(u32 index, float custom_tex_scale
     // Correct this with the upscaling factor of custom textures.
     s32 lod_offset = std::log2(g_framebuffer_manager->GetEFBScale() / custom_tex_scale) * 256.f;
     state.tm0.lod_bias = std::clamp<s32>(state.tm0.lod_bias + lod_offset, -32768, 32767);
-
-    // Anisotropic also pushes mips farther away so it cannot be used either
-    state.tm0.anisotropic_filtering = false;
```

Comment:
This needs to be kept I think?  Or at least, whatever is equivalent (I think 0?).

Reply:
It currently takes on a value of 0 or whatever `SamplerState::Generate` produces. I'm not really sure what's accurate. The change came from @Pokechu22.

Reply:
Yeah but here we are wanting to override the value of ansiotropic filtering, to ensure that the arbitrary mipmaps works properly.  So while it _could_ be 0, it might not be, which is why we need to explicitly set it.

Reply:
I don't remember why I removed this. The comment is correct in that (at least on real hardware; I'm not sure about modern graphics APIs) anisotropic filtering does push mips away, and probably would interact weirdly with arbitrary mipmap detection.

Reply:
I've restored the removed line.